From 410f5ab77b2222160ef86ea66f7561941402325f Thu, 27 Oct 2011 11:03:37 +0200
From: Bram Goffings <bramgoffings@gmail.com>
Date: Thu, 27 Oct 2011 11:03:17 +0200
Subject: [PATCH] - fixed panels layouts

diff --git a/alpha/css/alpha-mobile.css b/alpha/css/alpha-mobile.css
index e040671..2bc138b 100644
--- a/alpha/css/alpha-mobile.css
+++ b/alpha/css/alpha-mobile.css
@@ -8,4 +8,13 @@
   position: relative;
   margin-left: 10px;
   margin-right: 10px;
-}
\ No newline at end of file
+}
+
+/* Grid panels inside blocks need their margins negated. */
+.container-12 .block .omega-grid,
+.container-16 .block .omega-grid,
+.container-24 .block .omega-grid {
+  position: relative;
+  margin-left: -10px;
+  margin-right: -10px;
+}
diff --git a/omega/panels/layouts/omega_12/omega_12_onecol/omega-12-onecol.tpl.php b/omega/panels/layouts/omega_12/omega_12_onecol/omega-12-onecol.tpl.php
index 52d25e8..3660d18 100644
--- a/omega/panels/layouts/omega_12/omega_12_onecol/omega-12-onecol.tpl.php
+++ b/omega/panels/layouts/omega_12/omega_12_onecol/omega-12-onecol.tpl.php
@@ -1,4 +1,4 @@
-<div class="panel-display omega-12-onecol" <?php if (!empty($css_id)) { print "id=\"$css_id\""; } ?>>
+<div class="panel-display omega-grid omega-12-onecol" <?php if (!empty($css_id)) { print "id=\"$css_id\""; } ?>>
   <div class="panel-panel">
     <div class="inside"><?php print $content['middle']; ?></div>
   </div>
diff --git a/omega/panels/layouts/omega_12/omega_12_threecol_4_4_4/omega-12-threecol-4-4-4.tpl.php b/omega/panels/layouts/omega_12/omega_12_threecol_4_4_4/omega-12-threecol-4-4-4.tpl.php
index d3375ed..d74eec1 100644
--- a/omega/panels/layouts/omega_12/omega_12_threecol_4_4_4/omega-12-threecol-4-4-4.tpl.php
+++ b/omega/panels/layouts/omega_12/omega_12_threecol_4_4_4/omega-12-threecol-4-4-4.tpl.php
@@ -1,4 +1,4 @@
-<div class="panel-display omega-12-threecol-4-4-4" <?php if (!empty($css_id)) { print "id=\"$css_id\""; } ?>>
+<div class="panel-display omega-grid omega-12-threecol-4-4-4" <?php if (!empty($css_id)) { print "id=\"$css_id\""; } ?>>
   <div class="panel-panel grid-4">
     <div class="inside"><?php print $content['left']; ?></div>
   </div>
diff --git a/omega/panels/layouts/omega_12/omega_12_threecol_4_4_4_stacked/omega-12-threecol-4-4-4-stacked.tpl.php b/omega/panels/layouts/omega_12/omega_12_threecol_4_4_4_stacked/omega-12-threecol-4-4-4-stacked.tpl.php
index a2628ae..61b9066 100644
--- a/omega/panels/layouts/omega_12/omega_12_threecol_4_4_4_stacked/omega-12-threecol-4-4-4-stacked.tpl.php
+++ b/omega/panels/layouts/omega_12/omega_12_threecol_4_4_4_stacked/omega-12-threecol-4-4-4-stacked.tpl.php
@@ -1,4 +1,4 @@
-<div class="panel-display omega-12-threecol-4-4-4" <?php if (!empty($css_id)) { print "id=\"$css_id\""; } ?>>
+<div class="panel-display omega-grid omega-12-threecol-4-4-4" <?php if (!empty($css_id)) { print "id=\"$css_id\""; } ?>>
   <div class="panel-panel grid-12">
     <div class="inside"><?php print $content['top']; ?></div>
   </div>
diff --git a/omega/panels/layouts/omega_12/omega_12_twocol_3_9/omega-12-twocol-3-9.tpl.php b/omega/panels/layouts/omega_12/omega_12_twocol_3_9/omega-12-twocol-3-9.tpl.php
index bf98b3e..c9702cb 100644
--- a/omega/panels/layouts/omega_12/omega_12_twocol_3_9/omega-12-twocol-3-9.tpl.php
+++ b/omega/panels/layouts/omega_12/omega_12_twocol_3_9/omega-12-twocol-3-9.tpl.php
@@ -1,4 +1,4 @@
-<div class="panel-display omega-12-twocol-3-9" <?php if (!empty($css_id)) { print "id=\"$css_id\""; } ?>>
+<div class="panel-display omega-grid omega-12-twocol-3-9" <?php if (!empty($css_id)) { print "id=\"$css_id\""; } ?>>
   <div class="panel-panel grid-3">
     <div class="inside"><?php print $content['left']; ?></div>
   </div>
diff --git a/omega/panels/layouts/omega_12/omega_12_twocol_6_6/omega-12-twocol-6-6.tpl.php b/omega/panels/layouts/omega_12/omega_12_twocol_6_6/omega-12-twocol-6-6.tpl.php
index 6e60cef..b6088d6 100644
--- a/omega/panels/layouts/omega_12/omega_12_twocol_6_6/omega-12-twocol-6-6.tpl.php
+++ b/omega/panels/layouts/omega_12/omega_12_twocol_6_6/omega-12-twocol-6-6.tpl.php
@@ -1,4 +1,4 @@
-<div class="panel-display omega-12-twocol-6-6" <?php if (!empty($css_id)) { print "id=\"$css_id\""; } ?>>
+<div class="panel-display omega-grid omega-12-twocol-6-6" <?php if (!empty($css_id)) { print "id=\"$css_id\""; } ?>>
   <div class="panel-panel grid-6">
     <div class="inside"><?php print $content['left']; ?></div>
   </div>
diff --git a/omega/panels/layouts/omega_12/omega_12_twocol_6_6_bricks/omega-12-twocol-6-6-bricks.tpl.php b/omega/panels/layouts/omega_12/omega_12_twocol_6_6_bricks/omega-12-twocol-6-6-bricks.tpl.php
index 6a2e33c..3d077e4 100644
--- a/omega/panels/layouts/omega_12/omega_12_twocol_6_6_bricks/omega-12-twocol-6-6-bricks.tpl.php
+++ b/omega/panels/layouts/omega_12/omega_12_twocol_6_6_bricks/omega-12-twocol-6-6-bricks.tpl.php
@@ -1,4 +1,4 @@
-<div class="panel-display omega-12-twocol-6-6-bricks" <?php if (!empty($css_id)) { print "id=\"$css_id\""; } ?>>
+<div class="panel-display omega-grid omega-12-twocol-6-6-bricks" <?php if (!empty($css_id)) { print "id=\"$css_id\""; } ?>>
   <div class="panel-panel grid-12">
     <div class="inside"><?php print $content['top']; ?></div>
   </div>
diff --git a/omega/panels/layouts/omega_12/omega_12_twocol_6_6_stacked/omega-12-twocol-6-6-stacked.tpl.php b/omega/panels/layouts/omega_12/omega_12_twocol_6_6_stacked/omega-12-twocol-6-6-stacked.tpl.php
index 92aba8d..8727a82 100644
--- a/omega/panels/layouts/omega_12/omega_12_twocol_6_6_stacked/omega-12-twocol-6-6-stacked.tpl.php
+++ b/omega/panels/layouts/omega_12/omega_12_twocol_6_6_stacked/omega-12-twocol-6-6-stacked.tpl.php
@@ -1,4 +1,4 @@
-<div class="panel-display omega-12-twocol-6-6-stacked" <?php if (!empty($css_id)) { print "id=\"$css_id\""; } ?>>
+<div class="panel-display omega-grid omega-12-twocol-6-6-stacked" <?php if (!empty($css_id)) { print "id=\"$css_id\""; } ?>>
   <div class="panel-panel grid-12">
     <div class="inside"><?php print $content['top']; ?></div>
   </div>
diff --git a/omega/panels/layouts/omega_12/omega_12_twocol_9_3/omega-12-twocol-9-3.tpl.php b/omega/panels/layouts/omega_12/omega_12_twocol_9_3/omega-12-twocol-9-3.tpl.php
index 0edc733..1783d66 100644
--- a/omega/panels/layouts/omega_12/omega_12_twocol_9_3/omega-12-twocol-9-3.tpl.php
+++ b/omega/panels/layouts/omega_12/omega_12_twocol_9_3/omega-12-twocol-9-3.tpl.php
@@ -1,4 +1,4 @@
-<div class="panel-display omega-12-twocol-9-3" <?php if (!empty($css_id)) { print "id=\"$css_id\""; } ?>>
+<div class="panel-display omega-grid omega-12-twocol-9-3" <?php if (!empty($css_id)) { print "id=\"$css_id\""; } ?>>
   <div class="panel-panel grid-9">
     <div class="inside"><?php print $content['left']; ?></div>
   </div>
diff --git a/omega/panels/layouts/omega_16/omega_16_onecol/omega-16-onecol.tpl.php b/omega/panels/layouts/omega_16/omega_16_onecol/omega-16-onecol.tpl.php
index 7c23813..d6962ec 100644
--- a/omega/panels/layouts/omega_16/omega_16_onecol/omega-16-onecol.tpl.php
+++ b/omega/panels/layouts/omega_16/omega_16_onecol/omega-16-onecol.tpl.php
@@ -1,4 +1,4 @@
-<div class="panel-display omega-16-onecol" <?php if (!empty($css_id)) { print "id=\"$css_id\""; } ?>>
+<div class="panel-display omega-grid omega-16-onecol" <?php if (!empty($css_id)) { print "id=\"$css_id\""; } ?>>
   <div class="panel-panel">
     <div class="inside"><?php print $content['middle']; ?></div>
   </div>
diff --git a/omega/panels/layouts/omega_16/omega_16_threecol_5_6_5/omega-16-threecol-5-6-5.tpl.php b/omega/panels/layouts/omega_16/omega_16_threecol_5_6_5/omega-16-threecol-5-6-5.tpl.php
index 837c641..171da3b 100644
--- a/omega/panels/layouts/omega_16/omega_16_threecol_5_6_5/omega-16-threecol-5-6-5.tpl.php
+++ b/omega/panels/layouts/omega_16/omega_16_threecol_5_6_5/omega-16-threecol-5-6-5.tpl.php
@@ -1,4 +1,4 @@
-<div class="panel-display omega-16-threecol-5-6-5" <?php if (!empty($css_id)) { print "id=\"$css_id\""; } ?>>
+<div class="panel-display omega-grid omega-16-threecol-5-6-5" <?php if (!empty($css_id)) { print "id=\"$css_id\""; } ?>>
   <div class="panel-panel grid-5">
     <div class="inside"><?php print $content['left']; ?></div>
   </div>
diff --git a/omega/panels/layouts/omega_16/omega_16_threecol_5_6_5_stacked/omega-16-threecol-5-6-5-stacked.tpl.php b/omega/panels/layouts/omega_16/omega_16_threecol_5_6_5_stacked/omega-16-threecol-5-6-5-stacked.tpl.php
index 8c41ad9..43ba256 100644
--- a/omega/panels/layouts/omega_16/omega_16_threecol_5_6_5_stacked/omega-16-threecol-5-6-5-stacked.tpl.php
+++ b/omega/panels/layouts/omega_16/omega_16_threecol_5_6_5_stacked/omega-16-threecol-5-6-5-stacked.tpl.php
@@ -1,4 +1,4 @@
-<div class="panel-display omega-16-threecol-5-6-5" <?php if (!empty($css_id)) { print "id=\"$css_id\""; } ?>>
+<div class="panel-display omega-grid omega-16-threecol-5-6-5" <?php if (!empty($css_id)) { print "id=\"$css_id\""; } ?>>
   <div class="panel-panel grid-16">
     <div class="inside"><?php print $content['top']; ?></div>
   </div>
diff --git a/omega/panels/layouts/omega_16/omega_16_twocol_12_4/omega-16-twocol-12-4.tpl.php b/omega/panels/layouts/omega_16/omega_16_twocol_12_4/omega-16-twocol-12-4.tpl.php
index 1a0ad5d..67f6e9e 100644
--- a/omega/panels/layouts/omega_16/omega_16_twocol_12_4/omega-16-twocol-12-4.tpl.php
+++ b/omega/panels/layouts/omega_16/omega_16_twocol_12_4/omega-16-twocol-12-4.tpl.php
@@ -1,4 +1,4 @@
-<div class="panel-display omega-16-twocol-12-4" <?php if (!empty($css_id)) { print "id=\"$css_id\""; } ?>>
+<div class="panel-display omega-grid omega-16-twocol-12-4" <?php if (!empty($css_id)) { print "id=\"$css_id\""; } ?>>
   <div class="panel-panel grid-12">
     <div class="inside"><?php print $content['left']; ?></div>
   </div>
diff --git a/omega/panels/layouts/omega_16/omega_16_twocol_4_12/omega-16-twocol-4-12.tpl.php b/omega/panels/layouts/omega_16/omega_16_twocol_4_12/omega-16-twocol-4-12.tpl.php
index 762a045..eaae2d3 100644
--- a/omega/panels/layouts/omega_16/omega_16_twocol_4_12/omega-16-twocol-4-12.tpl.php
+++ b/omega/panels/layouts/omega_16/omega_16_twocol_4_12/omega-16-twocol-4-12.tpl.php
@@ -1,8 +1,8 @@
-<div class="panel-display omega-16-twocol-4-12" <?php if (!empty($css_id)) { print "id=\"$css_id\""; } ?>>
+<div class="panel-display omega-grid omega-16-twocol-4-12" <?php if (!empty($css_id)) { print "id=\"$css_id\""; } ?>>
   <div class="panel-panel grid-4">
     <div class="inside"><?php print $content['left']; ?></div>
   </div>
   <div class="panel-panel grid-12">
     <div class="inside"><?php print $content['right']; ?></div>
   </div>
-</div>
\ No newline at end of file
+</div>
diff --git a/omega/panels/layouts/omega_16/omega_16_twocol_8_8/omega-16-twocol-8-8.tpl.php b/omega/panels/layouts/omega_16/omega_16_twocol_8_8/omega-16-twocol-8-8.tpl.php
index 525b045..39e2382 100644
--- a/omega/panels/layouts/omega_16/omega_16_twocol_8_8/omega-16-twocol-8-8.tpl.php
+++ b/omega/panels/layouts/omega_16/omega_16_twocol_8_8/omega-16-twocol-8-8.tpl.php
@@ -1,4 +1,4 @@
-<div class="panel-display omega-16-twocol-8-8" <?php if (!empty($css_id)) { print "id=\"$css_id\""; } ?>>
+<div class="panel-display omega-grid omega-16-twocol-8-8" <?php if (!empty($css_id)) { print "id=\"$css_id\""; } ?>>
   <div class="panel-panel grid-8">
     <div class="inside"><?php print $content['left']; ?></div>
   </div>
diff --git a/omega/panels/layouts/omega_16/omega_16_twocol_8_8_bricks/omega-16-twocol-8-8-bricks.tpl.php b/omega/panels/layouts/omega_16/omega_16_twocol_8_8_bricks/omega-16-twocol-8-8-bricks.tpl.php
index 15d9c3f..f77e87a 100644
--- a/omega/panels/layouts/omega_16/omega_16_twocol_8_8_bricks/omega-16-twocol-8-8-bricks.tpl.php
+++ b/omega/panels/layouts/omega_16/omega_16_twocol_8_8_bricks/omega-16-twocol-8-8-bricks.tpl.php
@@ -1,4 +1,4 @@
-<div class="panel-display omega-16-twocol-8-8-bricks" <?php if (!empty($css_id)) { print "id=\"$css_id\""; } ?>>
+<div class="panel-display omega-grid omega-16-twocol-8-8-bricks" <?php if (!empty($css_id)) { print "id=\"$css_id\""; } ?>>
   <div class="panel-panel grid-16">
     <div class="inside"><?php print $content['top']; ?></div>
   </div>
diff --git a/omega/panels/layouts/omega_16/omega_16_twocol_8_8_stacked/omega-16-twocol-8-8-stacked.tpl.php b/omega/panels/layouts/omega_16/omega_16_twocol_8_8_stacked/omega-16-twocol-8-8-stacked.tpl.php
index ecfe81a..83bf8aa 100644
--- a/omega/panels/layouts/omega_16/omega_16_twocol_8_8_stacked/omega-16-twocol-8-8-stacked.tpl.php
+++ b/omega/panels/layouts/omega_16/omega_16_twocol_8_8_stacked/omega-16-twocol-8-8-stacked.tpl.php
@@ -1,4 +1,4 @@
-<div class="panel-display omega-16-twocol-8-8-stacked" <?php if (!empty($css_id)) { print "id=\"$css_id\""; } ?>>
+<div class="panel-display omega-grid omega-16-twocol-8-8-stacked" <?php if (!empty($css_id)) { print "id=\"$css_id\""; } ?>>
   <div class="panel-panel grid-16">
     <div class="inside"><?php print $content['top']; ?></div>
   </div>
diff --git a/omega/panels/layouts/omega_24/omega_24_onecol/omega-24-onecol.tpl.php b/omega/panels/layouts/omega_24/omega_24_onecol/omega-24-onecol.tpl.php
index fcab45c..98eea88 100644
--- a/omega/panels/layouts/omega_24/omega_24_onecol/omega-24-onecol.tpl.php
+++ b/omega/panels/layouts/omega_24/omega_24_onecol/omega-24-onecol.tpl.php
@@ -1,4 +1,4 @@
-<div class="panel-display omega-24-onecol" <?php if (!empty($css_id)) { print "id=\"$css_id\""; } ?>>
+<div class="panel-display omega-grid omega-24-onecol" <?php if (!empty($css_id)) { print "id=\"$css_id\""; } ?>>
   <div class="panel-panel">
     <div class="inside"><?php print $content['middle']; ?></div>
   </div>
diff --git a/omega/panels/layouts/omega_24/omega_24_threecol_8_8_8/omega-24-threecol-8-8-8.tpl.php b/omega/panels/layouts/omega_24/omega_24_threecol_8_8_8/omega-24-threecol-8-8-8.tpl.php
index 8fdfe96..ade4de3 100644
--- a/omega/panels/layouts/omega_24/omega_24_threecol_8_8_8/omega-24-threecol-8-8-8.tpl.php
+++ b/omega/panels/layouts/omega_24/omega_24_threecol_8_8_8/omega-24-threecol-8-8-8.tpl.php
@@ -1,4 +1,4 @@
-<div class="panel-display omega-24-threecol-8-8-8" <?php if (!empty($css_id)) { print "id=\"$css_id\""; } ?>>
+<div class="panel-display omega-grid omega-24-threecol-8-8-8" <?php if (!empty($css_id)) { print "id=\"$css_id\""; } ?>>
   <div class="panel-panel grid-8">
     <div class="inside"><?php print $content['left']; ?></div>
   </div>
diff --git a/omega/panels/layouts/omega_24/omega_24_threecol_8_8_8_stacked/omega-24-threecol-8-8-8-stacked.tpl.php b/omega/panels/layouts/omega_24/omega_24_threecol_8_8_8_stacked/omega-24-threecol-8-8-8-stacked.tpl.php
index 0ed1efd..10a1adf 100644
--- a/omega/panels/layouts/omega_24/omega_24_threecol_8_8_8_stacked/omega-24-threecol-8-8-8-stacked.tpl.php
+++ b/omega/panels/layouts/omega_24/omega_24_threecol_8_8_8_stacked/omega-24-threecol-8-8-8-stacked.tpl.php
@@ -1,4 +1,4 @@
-<div class="panel-display omega-24-threecol-8-8-8-stacked" <?php if (!empty($css_id)) { print "id=\"$css_id\""; } ?>>
+<div class="panel-display omega-grid omega-24-threecol-8-8-8-stacked" <?php if (!empty($css_id)) { print "id=\"$css_id\""; } ?>>
   <div class="panel-panel grid-24">
     <div class="inside"><?php print $content['top']; ?></div>
   </div>
diff --git a/omega/panels/layouts/omega_24/omega_24_twocol_12_12/omega-24-twocol-12-12.tpl.php b/omega/panels/layouts/omega_24/omega_24_twocol_12_12/omega-24-twocol-12-12.tpl.php
index f5fb79e..3ccf36c 100644
--- a/omega/panels/layouts/omega_24/omega_24_twocol_12_12/omega-24-twocol-12-12.tpl.php
+++ b/omega/panels/layouts/omega_24/omega_24_twocol_12_12/omega-24-twocol-12-12.tpl.php
@@ -1,4 +1,4 @@
-<div class="panel-display omega-24-twocol-12-12" <?php if (!empty($css_id)) { print "id=\"$css_id\""; } ?>>
+<div class="panel-display omega-grid omega-24-twocol-12-12" <?php if (!empty($css_id)) { print "id=\"$css_id\""; } ?>>
   <div class="panel-panel grid-12">
     <div class="inside"><?php print $content['left']; ?></div>
   </div>
diff --git a/omega/panels/layouts/omega_24/omega_24_twocol_12_12_bricks/omega-24-twocol-12-12-bricks.tpl.php b/omega/panels/layouts/omega_24/omega_24_twocol_12_12_bricks/omega-24-twocol-12-12-bricks.tpl.php
index 3216920..3b1b72c 100644
--- a/omega/panels/layouts/omega_24/omega_24_twocol_12_12_bricks/omega-24-twocol-12-12-bricks.tpl.php
+++ b/omega/panels/layouts/omega_24/omega_24_twocol_12_12_bricks/omega-24-twocol-12-12-bricks.tpl.php
@@ -1,4 +1,4 @@
-<div class="panel-display omega-24-twocol-12-12-bricks" <?php if (!empty($css_id)) { print "id=\"$css_id\""; } ?>>
+<div class="panel-display omega-grid omega-24-twocol-12-12-bricks" <?php if (!empty($css_id)) { print "id=\"$css_id\""; } ?>>
   <div class="panel-panel grid-24">
     <div class="inside"><?php print $content['top']; ?></div>
   </div>
diff --git a/omega/panels/layouts/omega_24/omega_24_twocol_12_12_stacked/omega-24-twocol-12-12-stacked.tpl.php b/omega/panels/layouts/omega_24/omega_24_twocol_12_12_stacked/omega-24-twocol-12-12-stacked.tpl.php
index a939b26..fd6d5cf 100644
--- a/omega/panels/layouts/omega_24/omega_24_twocol_12_12_stacked/omega-24-twocol-12-12-stacked.tpl.php
+++ b/omega/panels/layouts/omega_24/omega_24_twocol_12_12_stacked/omega-24-twocol-12-12-stacked.tpl.php
@@ -1,4 +1,4 @@
-<div class="panel-display omega-24-twocol-12-12-stacked" <?php if (!empty($css_id)) { print "id=\"$css_id\""; } ?>>
+<div class="panel-display omega-grid omega-24-twocol-12-12-stacked" <?php if (!empty($css_id)) { print "id=\"$css_id\""; } ?>>
   <div class="panel-panel grid-24">
     <div class="inside"><?php print $content['top']; ?></div>
   </div>
diff --git a/omega/panels/layouts/omega_24/omega_24_twocol_18_6/omega-24-twocol-18-6.tpl.php b/omega/panels/layouts/omega_24/omega_24_twocol_18_6/omega-24-twocol-18-6.tpl.php
index 20a597b..83fde4c 100644
--- a/omega/panels/layouts/omega_24/omega_24_twocol_18_6/omega-24-twocol-18-6.tpl.php
+++ b/omega/panels/layouts/omega_24/omega_24_twocol_18_6/omega-24-twocol-18-6.tpl.php
@@ -1,4 +1,4 @@
-<div class="panel-display omega-24-twocol-6-18" <?php if (!empty($css_id)) { print "id=\"$css_id\""; } ?>>
+<div class="panel-display omega-grid omega-24-twocol-6-18" <?php if (!empty($css_id)) { print "id=\"$css_id\""; } ?>>
   <div class="panel-panel grid-18">
     <div class="inside"><?php print $content['left']; ?></div>
   </div>
diff --git a/omega/panels/layouts/omega_24/omega_24_twocol_6_18/omega-24-twocol-6-18.tpl.php b/omega/panels/layouts/omega_24/omega_24_twocol_6_18/omega-24-twocol-6-18.tpl.php
index 3dbd34f..6efb801 100644
--- a/omega/panels/layouts/omega_24/omega_24_twocol_6_18/omega-24-twocol-6-18.tpl.php
+++ b/omega/panels/layouts/omega_24/omega_24_twocol_6_18/omega-24-twocol-6-18.tpl.php
@@ -1,4 +1,4 @@
-<div class="panel-display omega-24-twocol-6-18" <?php if (!empty($css_id)) { print "id=\"$css_id\""; } ?>>
+<div class="panel-display omega-grid omega-24-twocol-6-18" <?php if (!empty($css_id)) { print "id=\"$css_id\""; } ?>>
   <div class="panel-panel grid-6">
     <div class="inside"><?php print $content['left']; ?></div>
   </div>
